public/interviews.js

```diff
@@ -1168,160 +1168,203 @@ filter: drop-shadow(0px 0px 5px rgba(0, 0, 0, 0.25));"/>
 
     title: "Christian Fischer – Infected Chaos",
 
-    text: `The austrian-german metalband **INFECTED CHAOS** rose from the ashes of former band **Chaosreign** in 2013 and directly started to write their own songs. After intense road-tests – among others supporting the legendary **Possessed** – the band entered the renowned **Iguana Studio** and could release its debut **“The Wake of Ares”** in 2015.
+    text: `The austrian-german metalband [INFECTED CHAOS](https://www.facebook.com/InfectedChaos/) rose from the ashes of former band [Chaosreign](https://www.metal-archives.com/bands/Chaosreign/3540308719) in 2013 and directly started to write their own songs.
+  
+  After intense road-tests – among others supporting the legendary **Possessed** – the band entered the renowned [Iguana Studio](https://www.facebook.com/iguanastudios/?ref=page_internal) and could release its debut **“The Wake of Ares”** in 2015.
 
-Only two years later the successor **“Killing Creator”** was unleashed and led the band through the half of Europe. Well known festivals such as **MetalDays**, **Winterdays of Metal** and **Aaargh!-Festival** were *haunted*. *
-*In 2021 **“Dead Aesthetics”** was finally recorded and released by **INFECTED CHAOS**.
+Only two years later the successor **“Killing Creator”** was unleashed and led the band through the half of Europe. 
+
+Well known festivals such as [MetalDays](https://www.metaldays.net/),[Winterdays of Metal](https://www.instagram.com/winterdaysofmetal/) and [Aaargh!-Festival](https://www.aaarghfestival.de/) were *haunted*.
+
+In 2021 **“Dead Aesthetics”** was finally recorded and released by [INFECTED CHAOS](https://www.facebook.com/InfectedChaos/).
 
 At the moment the band is housing their own studio to create their next release.
 
 
-*As always, a huge **Thank You** to the band for their time, openness and availability!*
+As always, a huge **Thank You** to the band for their time, openness and availability!
+
+<hr style="background-color: #dfdfdf; height: 2px;box-shadow: 0px -5px 10px rgba(0, 0, 0, 0.25); width: 100px;
+filter: drop-shadow(0px 0px 5px rgba(0, 0, 0, 0.25));"/>
+
+
+<p style="background-color: #dfdfdf;"><strong>All Access Pass – Greetings and thank you for taking the time to talk to All Access Pass. It is a pleasure to be able to run this interview! 
```

Comment:
We usually don't use inline styling, also because it is very repetitive. You could create a Styled component "StyledParagraph" and reuse it everywhere in this document.

Reply:
Yes the styling changes and text edits are all part of the US12. I made a general US for style and legibility. And following one session with one of our teachers we decided to install Markdown for React and based on that individually edit the articles. The texts are quite long and do not really follow a pattern or structure that can be 100% added to all of the interviews. By using the markdown styling I can adjust all texts to break whenever the paragraph makes sense and add links to certain keywords and tags.